react-redux/src/main/scala/demo/advanced/ExpenseContainer.scala

```diff
@@ -61,11 +62,13 @@ object ExpenseContainer {
     (dispatch: Dispatch[AppActions]) =>
       js.Dynamic.literal(
         startAddExpense = bindActionCreators(addExpense, dispatch.asInstanceOf[Dispatch[AnyAction]]),
```

Comment:
is it possible to scalafy this a bit? Something like
```scala

trait LinkDispatchProps { // extends js.Object if it has to
  def startAddExpense(e: Expense): Unit
  // ...
}

new LinkDispatchProps { 
  override def startAddExpense(e: Expense): Unit = dispatch(setExpenses(e))
}
```

Reply:
I'm going to try it

Reply:
I get this error :(

`Uncaught TypeError: Cannot read property 'dispatch$1' of undefined`

Reply:
There is both a commit and this comment, does that mean you got it to work in the end?

Reply:
Yes, I was working on this. I think it looks better now